person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id # getters for @id, @name, and @age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = generate_id # generate a unique ID for the person
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ OPTIONAL] I recommend here to add parenthesis here when you are calling a method so kindly add them to be somthing like this 💯✨🔥✅

`def can_use_services? 
    is_of_age?() || @parent_permission
  end
`

Reply:
thanks @Reem-lab